### PR TITLE
v1.19.0

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- New: Review Schema Block.
- Improvement: Better options for delete/edit/duplicate action of each item in Price list block.
- Improvement: Ability to modify breakpoints for Dynamic CSS.
- Improvement: Table of Content - Exclude specific headings from Table of Contents block.
- Improvement: Table of Content - Fixed issue with the scroll to heading when the heading has ellipsis dots in them.
- Fix: Post Carousel - Fixed carousel not being displayed on RTL websites.
- Fix: Info Box - Link to complete box target issue with "Open In the Same Tab" fixed.

### Types of changes

- New feature ( New block, improvements, fixes .etc )

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/price-list-child/attributes.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const { __ } = wp.i18n
+
+
+const attributes = { 
+	block_id: {
+		type: "string"
+	},
+	classMigrate: {
```
Comment (vrundakansara) [inline src/blocks/price-list-child/attributes.js:8]: Do we need this `classMigrate` variable?

**Review by vrundakansara — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]

File: src/blocks/buttons/block.js [hunk 2]
```diff
@@ -35,9 +35,32 @@ registerBlockType( "uagb/buttons", {
 	supports: {
 		anchor: true,
 	},
+	example: {},
```
Comment (RajkiranBagal) [inline src/blocks/buttons/block.js:38]: @AbhijitNage123  I guess we can remove this.

File: src/blocks/faq/block.js [hunk 3]
```diff
@@ -62,8 +62,31 @@ registerBlockType( "uagb/faq", {
...
+						name: "uagb/faq-child",
+						attributes: { question: "What is FAQ?", answer: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat." },
+					},
+				],
+			}
+		],
+	},
 	attributes,
 	edit,
+	example: {},
```
Comment (RajkiranBagal) [inline src/blocks/faq/block.js:89]: @AbhijitNage123  We can remove this

File: src/blocks/review/attributes.js [hunk 4]
```diff
@@ -0,0 +1,466 @@
...
+  const { __ } = wp.i18n
+
+ const attributes = {
+	ID: {
+		type: "string",
+		default: "",
+	},
+	itemType: {
+		type: "string",
+		default: "Product",
```
Comment (RajkiranBagal) [inline src/blocks/review/attributes.js:13]: This should be Translatable

File: src/blocks/review/attributes.js [hunk 5]
```diff
@@ -0,0 +1,466 @@
...
+		type: "array",
+		default: [{ label: "", value: 0 }],
+	},
+	starCount: {
+		type: "number",
+		default: 5,
+	},
+	summaryTitle: {
+		type: "string",
+		default: "Summary",
```
Comment (RajkiranBagal) [inline src/blocks/review/attributes.js:130]: This should be Translatable

File: src/blocks/review/attributes.js [hunk 6]
```diff
@@ -0,0 +1,466 @@
...
+		type: "string",
+		default: "US$65"
+	},
+	pricetext: {
+		type: "string",
+		default: "Price"
+	},
+	availabilityvalue: {
+		type: "string",
+		default: "Instock"
```
Comment (RajkiranBagal) [inline src/blocks/review/attributes.js:314]: Check all such strings that should be translatable in this file.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**